doc/api/fs.markdown

```diff
@@ -1157,6 +1157,17 @@ reliably or at all.
 You can still use `fs.watchFile`, which uses stat polling, but it is slower and
 less reliable.
 
+#### Inodes
+
+<!--type=misc-->
+
+On Linux and OS X systems, `fs.watch` resolves the watched path to an inode. If
+the watched path is deleted and then created so that it has a new inode, you
```

Comment:
In general I'd like to see us move away from the informal use of "you" as a best practice. It will take a while to eliminate past usage but I'd prefer not to introduce new instances of it :-)